<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSetting</class>
 <widget class="QDialog" name="SystemSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#SystemSetting {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 148, 255, 255), stop:0.414773 rgba(168, 232, 255, 255), stop:0.738636 rgba(219, 242, 255, 255), stop:1 rgba(212, 255, 255, 255));}</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_recon">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Arial&quot;;

</string>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Reconstruction Enable</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>60</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLabel" name="label_wavelength">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="locale">
       <locale language="English" country="Canada"/>
      </property>
      <property name="text">
       <string>Wavelength (nm)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_RI">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="locale">
       <locale language="English" country="Canada"/>
      </property>
      <property name="text">
       <string>Refractive Index</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="pixel_size">
      <property name="locale">
       <locale language="English" country="Canada"/>
      </property>
      <property name="minimum">
       <double>0.100000000000000</double>
      </property>
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>1.850000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="magnification">
      <property name="locale">
       <locale language="English" country="Canada"/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>150</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QDoubleSpinBox" name="wave_length">
      <property name="locale">
       <locale language="English" country="Canada"/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>350.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1500.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="stepType">
       <enum>QAbstractSpinBox::DefaultStepType</enum>
      </property>
      <property name="value">
       <double>640.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_mag">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="locale">
       <locale language="English" country="Canada"/>
      </property>
      <property name="text">
       <string>Magnification</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_pixel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="locale">
       <locale language="English" country="Canada"/>
      </property>
      <property name="text">
       <string>Pixel Size (um)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="diffraction_index">
      <property name="locale">
       <locale language="English" country="Canada"/>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>1.520000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_confirm">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>160</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>160</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title_config">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Configuration Parameter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Title_recon">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Reconstruction Setting</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SpinBox_start_recon">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="minimum">
    <double>-2000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>2000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SpinBox_end_recon">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="minimum">
    <double>-2000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>2000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SpinBox_step_recon">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_start_recon">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Start(um):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_end_recon">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>End(um):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_step_recon">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>100</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Step(um):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FFT Filter Setting</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.200000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Hann</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Flat</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Size rate:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Filter type:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Noise &amp; Background</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_step_recon_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Sharpness Detection:  </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Fourier Analysis</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Statitic</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>What to Save</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_recon_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Height Map</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_recon_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Phase Map</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_recon_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Wrapped Phase</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_recon_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>280</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Refocused volume</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Leveling Method:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <item>
    <property name="text">
     <string>Gaussian Blur</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polynomial</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="text">
    <string>Filter size:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="Canada"/>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>20.000000000000000</double>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10.000000000000000</double>
   </property>
   <property name="value">
    <double>80.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
